packages/common/src/webrtc/Peer.ts

```diff
@@ -15,6 +15,14 @@ export default class Peer {
   private _constraints: { offerToReceiveAudio: boolean, offerToReceiveVideo: boolean }
   private _negotiating: boolean = false
 
+  resetNegotiating() {
```

Comment:
Generally, for code consistency, we write getters or any methods after the constructor.

_We can move this in some other PRs as well - your call._